### DISCUSSION TITLE
Installation failure, directory nonexistent (fixed)

### DISCUSSION THREAD

**Discussion by cloud9-kj** [google/timesketch#1877] [Q&A, quick solutions, support] [answered]
deploy_timesketch.sh: 19: [: Illegal number:
/usr/bin/docker-compose

Setting vm.max_map_count for Elasticsearch
Setting default config parameters..OK
Setting Elasticsearch memory allocation to 4GB
Fetching configuration files..deploy_timesketch.sh: 80: cannot create timesketch/etc/timesketch/timesketch.conf: Directory nonexistent
Is the URL still accessible? https://raw.githubusercontent.com/google/timesketch/master

**Comment by jaegeral** [comment, marked as answer]
According to the subject, this has been fixed. If something like that happens in the future, please open a github issue for it.

Thx
